### ISSUE TITLE
Warn on unknown property keys in `create()`, `fill()`, `update()`

### ISSUE THREAD

**Issue by alies-dev** [psalm/laravel-psalm-plugin#699]
Unknown array keys passed to `create()`, `fill()`, `update()`, and `forceFill()` should be reported.

```php
User::create(['nmae' => 'John']); // typo: 'nmae' instead of 'name'
$user->fill(['eamil' => 'test@test.com']); // unknown property
```

Larastan covers this via the `model-property<Model>` custom type.

**Implementation:** `AfterExpressionAnalysisInterface` — intercept the array argument to these methods, validate keys against known model properties (schema columns + casts + accessors + relationships).
